src/components/PokemonFiche.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+
+
+function PokemonFiche({name}) {
+
+    const [pokemonData, setPokemonData] = useState([]);
```

Comment:
`pokemonData` zal uiteindelijk een object zijn. Hierbij past een default waarde van `undefined` beter waardoor je ook een duidelijk verschil krijgt tussen of iets wel of niet is ingeladen
```suggestion
    const [pokemonData, setPokemonData] = useState();
```